### DISCUSSION TITLE
Cross-Origin Request Blocked : The Same Origin Policy disallows reading the remote resource at https://vitals.vercel-insights.com/v1/vitals.

### DISCUSSION THREAD

**Discussion by rahXephonz** [vercel/next.js#35214] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

    Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Enterprise
    Binaries:
      Node: 14.18.1
      npm: 6.14.15
      Yarn: 1.22.11
      pnpm: N/A
    Relevant packages:
      next: 12.0.8
      react: 17.0.2
      react-dom: 17.0.2

### What browser are you using? (if relevant)

Firefox

### How are you deploying your application? (if relevant)

I deploy it in Vercel and for my backend on Heroku

### Describe the Bug

The Same Origin Policy disallows reading the remote resource at https://vitals.vercel-insights.com/v1/vitals.

I see this error on console. Its weird my backend is allow all CORS and my transaction API infrontend its ok

### Expected Behavior

How to fix this error?

### To Reproduce

![image](https://user-images.githubusercontent.com/77242429/157630050-5a309ac1-b554-48db-9c45-ab43d8502423.png)

**Comment by rahXephonz** [comment]
Still not fixed🤔

**Reply by SawkaDev** [reply]
Any updates on this?
